# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../openoffice-org/cppunit-dmake.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./cppunit/cppunit-1.8.0-r1.patch.orig	2008-12-15 17:42:05.000000000 +0100
+++ ./cppunit/cppunit-1.8.0-r1.patch	2009-10-24 16:40:08.000000000 +0200
@@ -4289,9 +4289,9 @@
 +#*************************************************************************
 +
 +target:
-+	cd src$/win32 && $(MAKE) $(MFLAGS) $(CALLMACROS)
-+	cd src$/result && $(MAKE) $(MFLAGS) $(CALLMACROS)
-+	cd src$/cppunit && $(MAKE) $(MFLAGS) $(CALLMACROS)
++	cd src$/win32 && dmake $(MFLAGS) $(CALLMACROS)
++	cd src$/result && dmake $(MFLAGS) $(CALLMACROS)
++	cd src$/cppunit && dmake $(MFLAGS) $(CALLMACROS)
 Nur in misc/build/cppunit-1.8.0: makefile.mk.dummy.
 --- misc/cppunit-1.8.0/src/cppunit/Asserter.cpp	2002-03-27 20:04:02.000000000 +0100
 +++ misc/build/cppunit-1.8.0/src/cppunit/Asserter.cpp	2008-02-27 17:21:16.000000000 +0100
@@ -6296,9 +6296,9 @@
 +#*************************************************************************
 +
 +target:
-+	cd cppunit && $(MAKE) $(MFLAGS) $(CALLMACROS)
-+	cd result && $(MAKE) $(MFLAGS) $(CALLMACROS)
-+	cd win32 && $(MAKE) $(MFLAGS) $(CALLMACROS)
++	cd cppunit && dmake $(MFLAGS) $(CALLMACROS)
++	cd result && dmake $(MFLAGS) $(CALLMACROS)
++	cd win32 && dmake $(MFLAGS) $(CALLMACROS)
 Nur in misc/build/cppunit-1.8.0/src: makefile.mk.dummy.
 --- misc/cppunit-1.8.0/src/result/SynchronizedObject.cpp	2008-12-10 20:46:05.438373643 +0100
 +++ misc/build/cppunit-1.8.0/src/result/SynchronizedObject.cpp	2008-02-27 17:26:28.000000000 +0100
